### ISSUE TITLE
Standard weather states missing on weather entity

### ISSUE THREAD

**Issue by bremor** [bremor/bureau_of_meteorology#55]
did this today and its now updating every 10 minutes fine.
However the state seems to be two states, not sure how this can be addressed, but I want to trigger automations to move shades when it sunny, and the states seem to be things like
Mostly sunny.
sunny
exceptional
Showers. Windy.
Partly cloudy.
Early shower. Sunny afternoon.

I would prefer it to be the default weather states if possible

‘clear-night’
‘cloudy’
‘fog’
‘hail’
‘lightning’
‘lightning-rainy’
‘partlycloudy’
‘pouring’
‘rainy’
‘snowy’
‘snowy-rainy’
‘sunny’
‘windy’
‘windy-variant’
‘exceptional’

and the BOM attributes are in a BOM specificat variable

_Originally posted by @dave-schwarz in https://github.com/bremor/bureau_of_meteorology/issues/52#issuecomment-742413679_

**Comment by jamescaudle** [comment]
Hi, I am also having issues with the card not using the default weather states. Despite my best efforts, I seem unable to remap the state to one of the default states automatically only via manually changing the state in the developer panel. Can anyone else help?

**Comment by bremor** [comment]
@jamescaudle What is your use case for having the state as one of the default weather states? Would an attribute showing the default weather state work for you? I am trying to decide on a good solution because i'm sure some people would prefer the additional details of "Late rain and cold change. Windy." rather than just "Rainy".

**Comment by jamescaudle** [comment]
Hi @bremor, thanks for getting back to me. The first issue I noted was that the default weather card seems to generate the icon from the default weather states thus it is broken by using non standard states.

However I also use the default met.no based weather integration for other things such as adjusting the time my lights and blinds change depending on the weather rather than using ambient light sensors throughout the house. For example turning the evening lights on an hour earlier if it is overcast. It is difficult to do if there is no standardised state terminology.

Might is suggest that the standard terminology be used for the state (and thus would integrate with homeassistant perfectly) however make the extended descriptor such as "late rain and cold change" available as an attribute +/- separate sensor?

This would give everyone access to whichever they prefer and make coding of automations, icons and the like easier. Thanks for all your hard work! 🙂

**Comment by DavidFW1960** [comment]
I presume this request relates only to the say weather.home entity not the bom sensors the integration creates? If so then I don’t see this creating any issue because it will make the weather.home work with the inbuilt home assistant weather card and won’t impact anything else that I can see

**Comment by bremor** [comment]
Yeah it's just the 'weather.placename' entitiy. I will change this in the next release.

**Comment by jamescaudle** [comment]
For those interested I have managed to make some modifications to give access to both the standard state and the "short-text" descriptor on the default weather card:
![image](https://user-images.githubusercontent.com/71166080/103185145-11a34700-48f6-11eb-9121-8772c73e72fa.png)

I have hijacked the "attribution" attribute to allow the "short_text" descriptor to show below the current temperature. The changes also result in the weather entity and card using the standard home assistant weather states. Thus the BOM integration should now also work with third-party weather cards. Additionally, I can now use BOM data with my automations which rely on the home assistant standard weather state dictionary.

The changes I made within weather.py are as follows:
![image](https://user-images.githubusercontent.com/71166080/103185593-0b15cf00-48f8-11eb-8541-f2d2cb3e1524.png)

![image](https://user-images.githubusercontent.com/71166080/103185298-b6be1f80-48f6-11eb-9ea1-d2eb946bb427.png)
